### PR TITLE
Fix some broken links in the READMEs

### PR DESCRIPTION
**Pull request by tartley**
### Description

Fix some broken links in the README files referring to what is now old-examples.

Add a link from examples to old-examples, just to make them more discoverable.

Remove images from the old-examples/README, since the corresponding image files don't seem to exist any more.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ OpenGL 3.3 came out in February 2010. With **up to date drivers** you will be ab
 
 **Anywhere where OpenGL is supported.** ModernGL is capable of rendering using a [standalone_context] as well. Rendering to a window only requires a valid OpenGL context.
 
-[standalone_context]: https://github.com/cprogrammer1994/ModernGL/tree/master/examples/standalone
+[standalone_context]: https://github.com/cprogrammer1994/ModernGL/tree/master/examples/old-examples/standalone
```
Comment (tartley) [inline README.md]: Fixed broken link

File: examples/README.md [hunk 2]
```diff
@@ -38,4 +38,7 @@ Multiple instance rendering
 pip install -r requirements.txt
 ```
 
-## Work in progress...
```
Comment (tartley) [inline examples/README.md]: "Work in progress" notice serves no purpose.

File: examples/README.md [hunk 3]
```diff
@@ -38,4 +38,7 @@ Multiple instance rendering
...
+## Old examples
+
+Also see the [old examples](old-examples).
+
```
Comment (tartley) [inline examples/README.md]: Added a link to 'old examples' to make them more discoverable

File: examples/old-examples/README.md [hunk 4]
```diff
@@ -38,86 +38,67 @@ pip install PySDL2
 
 ## Contents
```
Comment (tartley) [inline examples/old-examples/README.md]: All the links below are fixed to refer to the 'old-examples' directory.

File: examples/old-examples/README.md [hunk 5]
```diff
@@ -38,86 +38,67 @@ pip install PySDL2
...
-  - [render_to_pillow_image](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/standalone/render_to_pillow_image.py)
-  - [render_to_pillow_image_multisample](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/standalone/render_to_pillow_image_multisample.py)
-  - [perspective_projection](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/standalone/perspective_projection.py)
-  - [window_coordinates](https://github.com/cprogrammer1994/ModernGL/tree/master/examples/standalone/window_coordinates.py)
+- [standalone](standalone)
+  - [01_hello_world](standalone/01_hello_world.py)
+  - [render_to_pillow_image](standalone/render_to_pillow_image.py)
+  - [render_to_pillow_image_multisample](standalone/render_to_pillow_image_multisample.py)
+  - [perspective_projection](standalone/perspective_projection.py)
+  - [window_coordinates](standalone/window_coordinates.py)
```
Comment (tartley) [inline examples/old-examples/README.md]: The following images were all broken links. I can't find exactly corresponding images in the repo any more, so I deleted this section.

**Comment by cprogrammer1994** [comment]
Thank you!

**Pull request merged**